packages/ui/components/top-banner/TopBanner.tsx

```diff
@@ -41,7 +42,7 @@ export function TopBanner(props: TopBannerProps) {
     <div
       data-testid="banner"
       className={classNames(
-        "flex min-h-[40px] w-full items-start justify-between gap-8 px-4 py-2 text-center lg:items-center",
+        `flex min-h-[${TOP_BANNER_HEIGHT}px] w-full items-start justify-between gap-8 px-4 py-2 text-center lg:items-center`,
```

Comment:
AFAIK this is not a recommended practice from the tailwindcss team. Here are the docs on it: https://tailwindcss.com/docs/content-configuration#dynamic-class-names